network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/waitlist/NetworkProtectionWaitlistImpl.kt

```diff
@@ -141,3 +147,36 @@ class NetPRemoteFeatureWrapper @Inject constructor(
         return netPRemoteFeature.self().isEnabled() && netPRemoteFeature.waitlist().isEnabled()
     }
 }
+
+internal interface DeviceTimezoneProvider {
+    fun getTimeZone(): TimeZone
+}
+
+@Retention(AnnotationRetention.BINARY)
+@Qualifier
+private annotation class InternalApi
+
+// visible for testing
+internal typealias USTimeZoneChecker = () -> Boolean
+
+@Module
+@ContributesTo(AppScope::class)
+class TimezoneProviderModule {
+    @Provides
+    @InternalApi
+    fun provideUsTimezoneChecker(): USTimeZoneChecker {
+        return {
+            val defaultTimeZone = TimeZone.getDefault()
+            val usTimeZoneIDs = arrayOf(
+                "America/New_York",
+                "America/Chicago",
+                "America/Denver",
+                "America/Los_Angeles",
+                "America/Anchorage",
+                "America/Honolulu",
```

Comment:
I was testing on my Pixel 6 and I get this value when selecting Unites States > Honolulu: `Pacific/Honolulu`